### DISCUSSION TITLE
Preview for unpublished page redirects to 'real' URL and fails

### DISCUSSION THREAD

**Discussion by rodrigo-c** [wagtail/wagtail#9638] [General]
Using wagtail 2.16.1, when I try to preview a page before publishing it, it goes first to https://example.com/admin/pages/12345/edit/preview/, and immediately redirects to https://example.com/page-url/ which gives an error, because the page isn't published.

**Comment by Scotchester** [comment]
This is probably an issue specific to your project. I recommend you initiate a conversation on one of our [support channels](https://wagtail.org/developers/).

**Reply by rodrigo-c** [reply]
Thanks!
